src/java/GrassPokemon.java

```diff
@@ -0,0 +1,46 @@
+public class GrassPokemon extends Pokemon {
+
+    private String attack;
+    private String leve1;
```

Comment:
Als je variabele in elke subklasse hetzelfde is, kan je je eigen afvragen of er subklassen nodig zijn. Ook als een variabele voor alle subklassen geld, kan je deze beter eenmalig in de super klasse zetten. Dan overerven al de subklasse deze variabele